releases/net472/KnownIssues/593963 - WPF Exceptions during AppDomain or process shutdown.md

```diff
@@ -0,0 +1,27 @@
+# WPF Exceptions during AppDomain or process shutdown
+
+## Symptoms
+In certain timing-sensitive WPF applications, exceptions may be seen during AppDomain or process shutdown.
+
+## Cause
+In .NET Framework 4.7.1 and earlier versions, WPF potentially creates a <xref:System.Windows.Threading.Dispatcher?displayProperty=nameWithType> on the .NET finalizer thread during AppDomain shutdown.  This was fixed in 
```

Comment:
Since this is a known issue rather than an app compat topic, it won't be published on docs, and the xref won't be resolved. This should be a conventional link: \[System.Windows.Threading.Dispatcher](https://docs.microsoft.com/en-us/dotnet/api/system.windows.threading.dispatcher)